src/sagas/giphySaga.js

```diff
@@ -0,0 +1,74 @@
+import { put, takeLatest, call, all } from 'redux-saga/effects';
+import {
+  FETCH_TRENDING_GIPHYS,
+  FETCH_TRENDING_GIPHYS_SUCCESS,
+  FETCH_GIPHYS_SEARCH,
+  FETCH_GIPHYS_BY_IDS,
+  FETCH_FAVORITE_GIPHYS_SUCCESS,
+  FETCH_GIPHY_DETAIL,
+  FETCH_GIPHY_DETAIL_SUCCESS
+} from '../actions/action_types';
+import {
+  fetchTrendingGiphysFromWS,
+  fetchSearchedGiphysFromWS,
+  fetchGiphysByIdsFromWS,
+  fetchGiphyDetailFromWS
+} from '../utils/giphyApi';
+
+function* fetchTrendingGiphys() {
+  const trendingGiphys = yield call(fetchTrendingGiphysFromWS);
+  yield put({
```

Comment:
You should use actionCreators here `yield put(fetchTrendingGiphysSuccess());` instead of creating the object inline.